libs/core/src/lib/input-group/input-group.component.html

```diff
@@ -1,25 +1,22 @@
 <div class="fd-input-group"
-     [ngClass]="{
-        'fd-input-group--after': placement === 'after',
-        'fd-input-group--before': placement === 'before',
-        'fd-input-group--inline': inline === true,
-        'fd-input-group--compact' : compact}">
+     [ngClass]="(inline ? 'fd-input-group--inline ' : ' ') +
+        (disabled ? 'is-disabled ' : ' ') +
+        (state ? 'is-' + state : ' ')">
```

Comment:
could also be written like this
[ngClass]="
{
        'fd-input-group--inline': inline,
        'is-disabled ' : disabled,
        is-' + state: state
    }

Reply:
It throws error, there is no way to add dynamically class as `key` in `ngClass` object